playwright-e2e/pages/dsm/home-page.ts

```diff
@@ -1,13 +1,63 @@
 import { expect, Locator, Page } from '@playwright/test';
+import { HomePageInterface } from 'pages/page-interface';
+import { DSMPageBase } from './page-base';
+import * as auth from 'authentication/auth-dsm';
+import Select from 'lib/widget/select';
+import { Navigation } from 'lib/component/dsm/navigation/navigation';
+import { StudyNav } from 'lib/component/dsm/navigation/enums/studyNav.enum';
+import ParticipantListPage from 'pages/dsm/participantList-page';
+import ParticipantPage from 'pages/dsm/participant-page';
 
 enum Titles {
   WELCOME = 'Welcome to the DDP Study Management System',
   SELECTED_STUDY = 'You have selected the ',
   STUDY = ' study.'
 }
 
-export default class HomePage {
-  constructor(private readonly page: Page) {}
+export default class HomePage extends DSMPageBase implements HomePageInterface {
+  private readonly navigation: Navigation;
+  private readonly participantList: ParticipantListPage;
+
+  constructor(page: Page) {
+    super(page);
+    this.navigation = new Navigation(page);
+    this.participantList = new ParticipantListPage(page);
+  }
+
+  /**
+   * Log into DSM application
+   * @param opts
+   */
+  async logIn(opts: { email?: string; password?: string; waitForNavigation?: boolean } = {}) {
+    await auth.login(this.page);
+  }
+
+  async selectStudy(study: string, page: Page) {
+    const studySelector = await new Select(page, { label: 'Select study' });
```

Comment:
You can select a study from the Navigation component, which is already implemented and you have already included it in the HomePage component (Line 18).
**Usage:**
`this.navigation.selectStudy()`

Reply:
Oh, anyway, this method is not used anywhere. Maybe remove it?